src/app/modules/static/artofreading.tpl.pug

```diff
@@ -23,10 +23,14 @@ div.staticPage
     br
     div
         | For Linux, assuming you have followed the
-        +link("/installers/linux") instructions for installing bloom on Linux
-        | , you just need one more command:
+        +link("/installers/linux") instructions for installing Bloom on Linux
+        | , you just need one more command to install Art of Reading version 3.3:
 
-        pre sudo apt-get install art-of-reading
+        pre     sudo apt-get install art-of-reading3
+        | For WeSay before 1.6 and Bloom before 4.0, the older version of Art of Reading (3.0.2) is required.  It has a different package name so that the older and newer versions of Art of Reading can coexist on the same system.  The command to install the older version is
```

Comment:
Alas we don't know when WeSay will update to libpalaso 3, so can you remove "before 1.6"? 

And below, remove the mention of Wesay.
Thanks.